e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -205,6 +205,15 @@ export const dashboardHeader = () => {
   return cy.findByTestId("dashboard-header");
 };
 
+export const createAndArchiveDashboard = ({ name, collection_id }) => {
+  cy.request("POST", "/api/dashboard/", {
+    name: name,
+    collection_id: collection_id,
+  }).then(({ body: { id: dashboardId } }) => {
+    cy.request("PUT", `/api/dashboard/${dashboardId}`, { archived: true });
+  });
+};
```

Comment:
nit, if you're only using this helper in one spec, I'd just keep it in that spec file. I doubt we'll be using this helper widely.